units/user-config.target

```diff
@@ -11,3 +11,9 @@ After=user-cloudinit@var-lib-coreos\x2dinstall-user_data.path
 
 Requires=user-cloudinit-proc-cmdline.service
 After=user-cloudinit-proc-cmdline.service
+
+Requires=user-config-vmware-tools.service
```

Comment:
given that user-config-vmware-tools.service is guarded by ConditionVirtualization=vmware, shouldn't it make it inexpensive in platforms where it really doesn't mean anything ?

Maybe I'm just misunderstanding systemd here, but I thought that was were the graph of agents would get trimmed. I'm not against having a vmware target, far from it (it'd probably be cleaner), but I was hoping performance-wise the difference would be minimal.